cmd/fleetctl/get.go

```diff
@@ -367,6 +392,24 @@ func getQueriesCommand() *cli.Command {
 	}
 }
 
+// userIsObserver returns whether the user is a global/team observer/observer+.
+// In the case of user belonging to multiple teams, a user is considered observer
+// if it is observer of all teams.
+func userIsObserver(user *fleet.User) (bool, error) {
+	if user.GlobalRole != nil {
```

Comment:
Maybe is worth it to add a check for nil here? `if user == nil {...}`?

Reply:
Good idea, see fbb8d8b56.